mesonbuild/build.py

```diff
@@ -964,7 +963,7 @@ def process_kwargs(self, kwargs, environment):
         if self.gnu_symbol_visibility != '':
             permitted = ['default', 'internal', 'hidden', 'protected', 'inlineshidden']
             if self.gnu_symbol_visibility not in permitted:
-                raise InvalidArguments('GNU symbol visibility arg %s not one of: %s',
+                raise InvalidArguments('GNU symbol visibility arg {} not one of: {}',
                                        self.symbol_visibility, ', '.join(permitted))
```

Comment:
This is missing `.format`. TBF the original was broken as well..

Reply:
Added the missing `.format`.

Reply:
😕 So.  Should there be a test to catch this?

Reply:
Given the way Python works, it's not really feasible.